rootfs/ingest/bin/replctl-ingest

```diff
@@ -43,21 +43,25 @@ from qserv.ingest import Ingester
 import qserv.util as util
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description="A python wrapper around "
-                                     "Qserv REST web service")
-
-    parser.add_argument("repl_url", type=str,
-                        help="Replication controller root URL",
+    parser = argparse.ArgumentParser(description="Launch Qserv replication service super-transactions "
+                                     "driving chunk files ingest")
+    parser.add_argument("--check", "-c", action='store_true',
+                        help="Check if chunk file ingestion has been performed successfully")
+    parser.add_argument("--chunk_queue-fraction", "-f", type=int, metavar="FRACTION",
```

Comment:
Not a good idea to mix `_` and `-` in the option specification. The name of the option `--chunk_queue-fraction` is a bit confusing. Choose one of those:
```
--chunk-queue-fraction
--chunk_queue_fraction
```
The former (with `-`)  is more standard.

Reply:
:+1: